### PR TITLE
Migrate to lutaml from plantuml

### PR DESCRIPTION
**Pull request by CAMOBAP**
- https://github.com/metanorma/metanorma-build-scripts/issues/62

### CODE REVIEW

**Comment by ronaldtse** [comment]
@CAMOBAP need to also change the CI job to show LutaML instead of PlantUML.

Screenshot 2021-03-16 at 6 35 32 PM

File: .github/workflows/make.yml [hunk 1]
```diff
@@ -9,46 +9,38 @@ on:
   pull_request:
 
 jobs:
-  test-macos:
-    name: Test on Ruby ${{ matrix.ruby }} macOS
-    runs-on: macos-latest
-    continue-on-error: ${{ matrix.experimental }}
+  make:
+    name: Test on PlantUML ${{ matrix.os }}
```
Comment (ronaldtse) [inline .github/workflows/make.yml:13]: LutaML not PlantUML

File: Makefile [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+endif
+
+PNG := $(patsubst views/%.lutaml,images/%.png,$(SRC))
 
 all: $(PNG)
 
 images/%.png: views/%.lutaml
 	lutaml -t png -o $@ $<
 
+views/%.lutaml: models/%.wsd
```
Comment (ronaldtse) [inline Makefile:23]: There is no `.wsd` file in this repository.

File: Makefile [hunk 3]
```diff
@@ -1,14 +1,33 @@
...
+SHELL := cmd
+else
+SHELL := /bin/bash
+endif
+
 SRC := $(wildcard views/*.lutaml)
-PNG := $(patsubst views/%.lutaml,images/%.png,$(SRC))
 
-SHELL := /bin/bash
+ifeq ($(SRC),)
```
Comment (ronaldtse) [inline Makefile:12]: No `.wsd` files in this repo.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: .github/workflows/make.yml (hunk 1 above)
Comment (CAMOBAP) [reply .github/workflows/make.yml:13]: Yep, I will fix it still, still testing

File: Makefile (hunk 2 above)
Comment (CAMOBAP) [reply Makefile:23]: Right, but I will add this Makefile to cimas to so we will use a single Makefile for all metanorma-model-* repositories

Comment (ronaldtse) [reply Makefile:23]: We will replace all .wsd files with .lutaml files.

Comment (CAMOBAP) [reply Makefile:23]: Ok, but while we didn't accomplish this, why not keep this target? Once we will have converted all wsd lutaml this target will be just ignored

File: .github/workflows/make.yml (hunk 1 above)
Comment (CAMOBAP) [reply .github/workflows/make.yml]: Fixed

**Comment by CAMOBAP** [comment]
@ronaldtse ready for review. If you still think that's better to remove `views/%.lutaml: models/%.wsd` please confirm, I will remove it, but in this case, other repositories without generated `lutaml` will failed

**Comment by ronaldtse** [comment]
OK let's keep it for now and we can remove those lines later.

**Review by ronaldtse — APPROVED** [review]

**Comment by CAMOBAP** [comment]
@ronaldtse thanks, I will start updating other repos

**Pull request merged**